src/main/java/io/jenkins/plugins/pipelinegraphview/utils/AbstractPipelineViewAction.java

```diff
@@ -57,6 +62,28 @@ public boolean isParameterized() {
         return property != null && !property.getParameterDefinitions().isEmpty();
     }
 
+    /**
+     * Handles the rebuild request and redirects to parameterized
+     * and non parameterized build when needed.
```

Comment:
this doesn't seem to be true? It just adds a notification with the result?

Reply:
Done